### PR TITLE
Added description to tmux

### LINKED ISSUE #170: tmux
Going to add description for tmux

### PR DESCRIPTION
**Pull request by qinyeli**
# Advanced HW 9 submission

Closes #170 

Name: Qinye Li
Uniqname: qinyeli

### CODE REVIEW

File: _commands/tools/tmux.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+&			kill the current window
+n			switch to the next window
+p			switch to the previous window
+f			promt a search in the current window
+,			rename the current window
+Up/Down		switch to the up/down pane
+Left/Right	switch to the eft/right pane
+```
+
+####
```
Comment (mterwill) [inline _commands/tools/tmux.md]: What's this line for?

**Comment by mterwill** [comment]
Add a `<!--more-->` line so it previews correctly on the website. See completed PRs for examples.

File: _commands/tools/tmux.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+```
+$ brew install tmux
+```
+Now you have installed tmux. 
+
+To start tmux, simply type `$ tmux`. To quit, type `$ exit`.
+
+#### Key binding
+`tmux` can be controlled using a key combination as a prefix key, (which is ctrl+b by default), followd by a command. The following are the most frequently used commands.
```
Comment (mterwill) [inline _commands/tools/tmux.md]: Use [keyboard keys](http://markdown2.github.io/site/syntax/#keyboard-keys)

File: _commands/tools/tmux.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+!			break the current pane out of the window
+x			kill the current pane
+c			create a new window
+&			kill the current window
+n			switch to the next window
+p			switch to the previous window
+f			promt a search in the current window
+,			rename the current window
+Up/Down		switch to the up/down pane
+Left/Right	switch to the eft/right pane
```
Comment (mterwill) [inline _commands/tools/tmux.md]: s/eft/left

**Comment by mterwill** [comment]
Add documentation for:
- `tmux at`
- `tmux ls`
- Note that `tmux.conf` is a thing

File: _commands/tools/tmux.md (hunk 2 above)
Comment (qinyeli) [inline _commands/tools/tmux.md]: What is a  line?

**Comment by themoh** [comment]
We'll be closing all pull requests at midnight tonight, so please have this done by then.

**Comment by mterwill** [comment]
Ccrraaaaazzzy 
screen shot 2016-03-27 at 7 42 32 pm

**Comment by mterwill** [comment]
There are a few minor things here, but looks good enough for me to merge. Thanks!

**Pull request merged**